lib/generators/activeadmin_addons/install/install_generator.rb

```diff
@@ -11,7 +11,8 @@ def setup_assets
         if use_webpacker?
           generate "activeadmin_addons:webpacker"
         else
-          generate "activeadmin_addons:assets"
+          puts "ActiveAdmin Addons requires ActiveAdmin installed with webpacker. "\
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Do not write to stdout. Use Rails's logger if you want to log.